index.html

```diff
@@ -0,0 +1,204 @@
+  <meta charset="UTF-8" />
+  <title>SD Global Conference</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="css/style.css" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous" />
+  <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet"
+    type="text/css" />
+</head>
+
+<body>
+  <div class="top-bar-section">
+    <div class="top-bar-container">
+      <ul class="top-bar">
+        <li>
+          <i class="fab fa-facebook-f"></i>
+        </li>
+        <li>
+          <i class="fab fa-twitter"></i>
+        </li>
+        <li>
+          <h4>English</h4>
+        </li>
+        <li>
+          <h4>My Page</h4>
+        </li>
+        <li>
+          <h4>Logout</h4>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <button title="hamburgermenu" type="button" class="hamburger">
+    <span class="bar"></span>
+  </button>
+  <nav class="mobile-nav">
+    <ul>
+      <li><a href="pages/about.html">About</a></li>
+      <li><a href="#program">Program</a></li>
+      <li><a href="#join">Join</a></li>
+      <li><a href="#sponsor">Sponsor</a></li>
+      <li><a href="#news">News</a></li>
+    </ul>
+  </nav>
+  <nav class="navbar navbar-expand-md navbar-light bg-light">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="index.html"><img src="images/logo1.png" alt="logo"></a>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <li class="nav-item">
+            <a class="nav-link active" aria-current="page" href="pages/about.html">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Program</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Join</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">News</a>
+          </li>
+        </ul>
+        <button class="btn btn-outline-success" type="submit">
+          SD Campaign
+        </button>
+      </div>
+    </div>
+  </nav>
+
+  <section class="first-section">
+    <div class="first-section-container">
+      <h3 class="second-heading">"Hello! Sharing world"</h3>
+      <h1 class="open-heading">SOFTWARE DEVELOPER GLOBAL SUMMIT 2021</h1>
+      <div class="first-section-paragraph">
+        <p class="first-paragraph">
+          O joyful celebration believing in the value of openness and
+          sharing,creating a positive change with people from over 80
+          xountries is taking place in October,in Korea.
```

Comment:
```suggestion
          countries is taking place in October, in Korea.
```
We all make typo errors.